app/components/primer/spinner_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use Primer::SpinnerComponent to let users know that content is being loaded.
+  class SpinnerComponent < Primer::Component
+
+    #
+    # @example 48|Default
+    #   <%= render(Primer::SpinnerComponent.new) %>
+    #
+    # @example 32|Small
+    #   <%= render(Primer::SpinnerComponent.new(size: 16)) %>
+    #
+    # @param size [Integer] Sets the size of the spinner
+    def initialize(size: 32, **kwargs)
+      @kwargs = kwargs
+      @kwargs[:tag] ||= :svg
```

Comment:
I believe this should _always_ be `svg`:

```suggestion
      @kwargs[:tag] = :svg
```